baseframe/forms/auto.py

```diff
@@ -16,7 +16,7 @@ class ConfirmDeleteForm(Form):
     cancel = SubmitField(__(u"Cancel"))
 
 
-def render_form(form, title, message='', formid='form', submit=__(u"Submit"), cancel_url=None, ajax=False):
+def render_form(form, title, message='', formid='form', submit=__(u"Submit"), cancel_url=None, ajax=False, theme=None):
```

Comment:
Instead of explicitly passing a theme, could we put this in the config somewhere?

1. In the app's `settings.py`
2. In Baseframe's `init_app`, which is typically called along with a `requires` clause, making maintenance easier. Baseframe can then save it in `app.config` and consult that on each request.
3. In the asset system itself, so that whenever a particular asset is requested, it sets a request-level flag that this function can consult.

My sense is the second method is the most straightforward and will cause the least future maintenance hassle.

Reply:
Passing `theme` in `init_app` and storing it in `app.config`